pymc/distributions/continuous.py

```diff
@@ -524,11 +525,11 @@ class Normal(Continuous):
 
     Parameters
     ----------
-    mu: float
-        Mean.
-    sigma: float
+    mu : float
```

Comment:
I think the type should be `numeric, array_like or TensorVariable` cc @Sayam753 @twiecki @michaelosthege